docs/modules/user-guide/partials/proc_migrating-commands.adoc

```diff
@@ -2,26 +2,26 @@
 = Migrating commands
 
 [role="_abstract"]
-This section describes how to migrate plug-ins to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.0. 
+This section describes how to migrate plug-ins to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.0.
 
 .Procedure
 
-. Command Groups: build,run,test,debug. 
-. Apply Command. 
-. Environment Varibables for a Specific Command. 
-. Renaming workdir into workingDir. 
-. Id and label for Composite Commands. 
-. Run exec Commands as Specific User. 
+. Command Groups: build, run, test, debug.
+. Apply Command.
```

Comment:
I think we need to update this document. This procedure doesnt really say anything about how to migrate, just list devfile v2 schema properties

Reply:
@maysunfaisal, think that'd be a separate issue we would open?